app/src/main/java/com/rage/clamber/Fragments/UserInfoFragment.java

```diff
@@ -71,12 +81,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         ButterKnife.bind(this, rootView);
 
         mainUser = getArguments().getParcelable(HomePage.ARG_USER);
-
+        if (mainUser != null) {
```

Comment:
is there a situation in which the main user would be `null`? I always support checking for null-ness, I'm just curious here